packages/router/src/lib/operations/auction.ts

```diff
@@ -229,31 +230,34 @@ export const newAuction = async (
     });
   }
 
-  const [senderBalance, receiverBalance] = await Promise.all([
-    txService.getBalance(sendingChainId, routerAddress),
-    txService.getBalance(receivingChainId, routerAddress),
-  ]);
-  logger.debug("Got balances", requestContext, methodContext, {
-    senderBalance: senderBalance.toString(),
-    receiverBalance: receiverBalance.toString(),
-  });
-
-  // Log if gas is low, but above min
-  const LOW_GAS = parseEther("0.1");
-  if (senderBalance.lt(LOW_GAS) || receiverBalance.lt(LOW_GAS)) {
-    logger.warn("Router has low gas", requestContext, methodContext, {
-      sendingChainId,
-      receivingChainId,
-      senderBalance: formatEther(senderBalance),
-      receiverBalance: formatEther(receiverBalance),
+  // No gasfees status check required for routerContractAddress
+  if (!isRouterContract) {
+    const [senderBalance, receiverBalance] = await Promise.all([
+      txService.getBalance(sendingChainId, routerAddress),
+      txService.getBalance(receivingChainId, routerAddress),
+    ]);
+    logger.debug("Got balances", requestContext, methodContext, {
+      senderBalance: senderBalance.toString(),
+      receiverBalance: receiverBalance.toString(),
     });
-  }
 
-  if (senderBalance.lt(sendingConfig.minGas) || receiverBalance.lt(receivingConfig.minGas)) {
-    throw new NotEnoughGas(sendingChainId, senderBalance, receivingChainId, receiverBalance, {
-      methodContext,
-      requestContext,
-    });
+    // Log if gas is low, but above min
+    const LOW_GAS = parseEther("0.1");
+    if (senderBalance.lt(LOW_GAS) || receiverBalance.lt(LOW_GAS)) {
+      logger.warn("Router has low gas", requestContext, methodContext, {
+        sendingChainId,
+        receivingChainId,
+        senderBalance: formatEther(senderBalance),
+        receiverBalance: formatEther(receiverBalance),
+      });
+    }
```

Comment:
This should be configured as some amount above the configured

Reply:
Why "some amount above"?